experimental/packages/opentelemetry-exporter-metrics-otlp-grpc/package.json

```diff
@@ -50,7 +50,7 @@
   "devDependencies": {
     "@babel/core": "7.16.0",
     "@opentelemetry/api": "^1.1.0",
-    "@opentelemetry/api-metrics": "0.27.0",
+    "@opentelemetry/api-metrics-wip": "0.27.0",
```

Comment:
I think the package should be renamed instead.

Reply:
We can do it in a separate PR: https://github.com/open-telemetry/opentelemetry-js/pull/2872

Reply:
In the SIG Meeting yesterday we came to the conclusion that we need to do it in this PR as we're having quite a few issues with dependencies. We'll also have to incorporate the changes from #2874. So in this PR I'll drop the `-wip` from the package names, which also entails updating the `otlp-transformer` package. I'm working on that right now.